services/fl-service/src/services/fl-service.ts

```diff
@@ -75,10 +78,57 @@ const validateConfig = (config: IOverridableConfig): IOverridableConfig => {
         validatedConfig.GLOBAL_ROUNDS = config.GLOBAL_ROUNDS;
     }
 
-    if(typeof config.IGNORE_RESULT_ERROR === "boolean") {
+    if (typeof config.IGNORE_RESULT_ERROR === "boolean") {
         validatedConfig.IGNORE_RESULT_ERROR = config.IGNORE_RESULT_ERROR;
     }
 
+    if (config.AGGREGATOR) {
+        if (FLAggregators.includes(config.AGGREGATOR)) {
+            validatedConfig.AGGREGATOR = config.AGGREGATOR;
+        } else {
+
+            throw new Error(`An unknown aggregator has been selected: ${config.AGGREGATOR}`);
+        }
+    }
+
+    if (config.AGGREGATION_WEIGHTS) {
+        if (config.AGGREGATION_WEIGHTS.constructor !== Object) {
```

Comment:
I've not come across this before but fair enough if it works! just to keep consistency though, could you use
```suggestion
        if (typeof config.AGGREGATION_WEIGHTS !== "object") {
```

Reply:
This issue with that implementation is that it won't catch an array if it is passed in.

Reply:
yep fair enough, just tested and you're right